pileup_CTCF.ipynb

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
<p>I'd say, for species with telocentric chromosomes (e.g. mouse)...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Ah, sure! Good point!